src/manifest.json

```diff
@@ -0,0 +1,42 @@
+{
+  "description": "WebExtension for Motrix download manager",
+  "manifest_version": 2,
+  "name": "Moxtrix WebExtension",
+  "version": "2.0.0",
+  "homepage_url": "https://github.com/gautamkrishnar/motrix-chrome-extension/",
+  "icons": {
+    "16": "assets/images/16.png",
+    "32": "assets/images/32.png",
+    "48": "assets/images/48.png",
+    "128": "assets/images/128.png"
+  },
+  "permissions": [
+    "downloads",
+    "storage",
+    "notifications"
+  ],
+  "browser_action": {
+    "browser_style": true,
+    "default_icon": {
+      "16": "assets/images/16.png",
+      "32": "assets/images/32.png",
+      "48": "assets/images/48.png",
+      "128": "assets/images/128.png"
+    },
+    "default_title": "Motrix WebExtension",
+    "default_popup": "popup.html"
+  },
+  "background": {
+    "scripts": [
+      "browser-polyfill.min.js",
+      "background.js"
+    ],
+    "persistent": true
+  },
+  "applications": {
+    "gecko": {
+      "id": "gautamkrishnar@github.com",
```

Comment:
Later will require permanent Application ID for Firefox to allow storage API to work.